package.json

```diff
@@ -21,7 +21,8 @@
     "version": "changeset version",
     "postversion": "pnpm run versionReadme",
     "versionReadme": "pnpx ts-node scripts/addChangelogBodyToReadmeFile.ts",
-    "taze": "taze major -wr"
+    "taze": "taze major -wr",
+    "prepare": "husky install"
```

Comment:
```suggestion
    "postinstall": "husky install"
```
this would do this automatically without need to remember about running `pnpm run prepare`, the question is how much time does it take to run install even if it's already installed?

Reply:
husky documentation suggest prepare, also prepare is executed during install, no need to execute pnpm run prepare manual

Reply:
nice!